ComponentKitTests/Scope/CKComponentScopeTests.mm

```diff
@@ -219,4 +219,28 @@ - (void)testComponentScopeHandleGlobalIdentifierIsTheSameBetweenSiblingsWithComp
   }
 }
 
+#pragma mark - Component scope override
+
+- (void)testComponentScopeCanBeOverridden
+{
+  CKComponentScopeRoot *root = [CKComponentScopeRoot rootWithListener:nil];
+  CKThreadLocalComponentScope threadScope(root, {});
+  CKThreadLocalComponentScope *threadScopePtr = &threadScope;
+
+  dispatch_semaphore_t sema = dispatch_semaphore_create(0);
+  dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
```

Comment:
Yeah, I'm not aware of another way of creating a component scope and _not_ making it also the thread scope. I could maybe stack them a afer b, and then overriding with b, but that might interplay in a strange way (and is not actually what we're planning to use it for).